providers/provider.js

```diff
@@ -260,11 +260,10 @@ function checkTransactionRequest(transaction) {
 }
 
 var formatTransactionReceiptLog = {
-    transactionLogIndex: checkNumber,
+    transactionIndex: checkNumber,
```

Comment:
this should be deleted as transactionIndex is declared further down in formatTransactionReceiptLog